core/imgread/common.cpp

```diff
@@ -23,6 +24,19 @@ Disc*(*drivers[])(const wchar* path)=
 
 u8 q_subchannel[96];
 
+typedef std::tuple<int, int, int> SenseErrorCode;
```

Comment:
why not simply use `typdef int SenseErrorCode[3]`?

Reply:
Or even a struct w/ asc, ascq and key?

Reply:
Struct is the best option, imo.